pow/randomx/src/lib.rs

```diff
@@ -242,21 +289,24 @@ mod tests {
 	use super::*;
 
 	#[test]
```

Comment:
Can we add a simple test here for the `reinit` function?

Reply:
:ok_hand: